packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1563,6 +1565,9 @@ function runTest<TChannelFactory extends IChannelFactory, TOperation extends Bas
 ): void {
 	const itFn = options.only.has(seed) ? it.only : options.skip.has(seed) ? it.skip : it;
 	itFn(`workload: ${model.workloadName} seed: ${seed}`, async function () {
+		if (options.beforeEach !== undefined) {
+			options.beforeEach();
```

Comment:
If you put the state of which ids are available on the state object under each client (similar to what we do for undo/redo tests), you won't need to make this change. It should also make debugging a bit nicer since the "currently known state of the client" is all in one place.